README.md

```diff
@@ -103,7 +103,7 @@ The first line adds a Hadamard gate to circuit object `c` which will operate on
 
 **Note**: Unlike C++ or Python, indexing in Julia starts from "1" and not "0"!
 
-The next step we want to take is to simulate our circuit. We do not need to transpile our circuit since our simulator can handle all gates, but for larger circuit you should consider transpilation to reduce the amount of work the simulator has to perform.
+The next step we want to take is to execute our circuit. Instead of submitting a job to a remote quantum service, we will use Snowflurry's built-in simulator. We do not need to transpile our circuit since our simulator can handle all gates, but for larger circuit you should consider transpilation to reduce the amount of work the simulator has to perform.
```

Comment:
I'm a little uneasy with "execute". Simulate() is the function we call for this

Reply:
Yeah, `execute` isn't great in this context... I do want to be consistent with what we have above, though... [lines 50-54](https://github.com/SnowflurrySDK/Snowflurry.jl/pull/326/files/226e26c3f87199399151d510d2777dbde4568d45#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L50-L54) say "Create -> Transpile -> Execute".

Paul pointed out that "Create -> Transpile -> Execute" is _a_ typical workflow but we're walking the user through something a bit different here.... We're kinda doing "Learn Snowflurry with Create then Simulate"... I'll patch the verbiage above to reflect what we're actually working through.